### ISSUE TITLE
Another schedulers can't be selected if anacron is available on the system

### ISSUE THREAD

**Issue by fandjelo** [emersion/bups#28]
In gtk.py:497 the call to get_job() for anacron returns some good value, so another schedulers, like systemctl are not checked anymore.

**Comment by emersion** [comment]
That's strange, this just selects _by default_ a scheduler. You're supposed to be able to select the others too. Are you sure systemd is installed on your system?

**Comment by fandjelo** [comment]
Yes, I am. I have both, cronie (anacron) and systemd, I'm using ArchLinux, so systemd is mandatory. May be I explained it wrong, it is not about selection in pull down menu, but about usage of scheduler. Just try following: 
1) Make sure, that you have anacron installed and so, have file /etc/anacron
2) Turn on one the systemd schedulers and leave the settings window
3) Come back to settings and see, that systemd is not shown as active, so you can't deactivate it anymore
